mne/utils.py

```diff
@@ -1545,6 +1545,8 @@ def _clean_names(names, remove_whitespace=False, before_dash=True):
             name = name.replace(' ', '')
         if '-' in name and before_dash:
             name = name.split('-')[0]
+        if '_virtual' in name:
```

Comment:
do we always append it to the end? If so, it might be safer to do `name.endswith()` and `name = name[:-8]`